lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart

```diff
@@ -54,7 +56,6 @@ class GuessSecretWordCubit
           Timer(const Duration(seconds: 2), () {
```

Comment:
Alterntively we can _emitData() above this and leave the return; where it is. But this solution is less code